sim/core/base_stats_auto_gen.go

```diff
@@ -9,42 +9,42 @@ import (
 	"github.com/wowsims/mop/sim/core/stats"
 )
 
-const ExpertisePerQuarterPercentReduction = 102.445740
-const HasteRatingPerHastePercent = 128.057160
-const CritRatingPerCritPercent = 179.280040
-const PhysicalHitRatingPerHitPercent = 102.445740
-const SpellHitRatingPerHitPercent = 102.445740
-const DodgeRatingPerDodgePercent = 265.078340
-const ParryRatingPerParryPercent = 265.078340
-const MasteryRatingPerMasteryPoint = 179.280040
+const ExpertisePerQuarterPercentReduction = 340.000000
+const HasteRatingPerHastePercent = 425.000000
+const CritRatingPerCritPercent = 600.000000
+const PhysicalHitRatingPerHitPercent = 340.000000
+const SpellHitRatingPerHitPercent = 340.000000
```

Comment:
Shouldn't the Expertise constant be 4x smaller since it's expressed in terms of quarter percent reductions in Dodge/Parry chance rather than full percent reductions for Hit?